Makefile

```diff
@@ -1,29 +1,6 @@
-NXDK_DIR = $(CURDIR)/../nxdk
```

Comment:
Must be documented in README, as it's very non-standard for todays nxdk (although it should be common-sense)